README.md

```diff
@@ -16,6 +16,8 @@ Set these environment variables if you need to change their defaults
 | ----------------------------- | ----------------------------------------------------------------- | -------------- |
 | TEMPORAL_GRPC_ENDPOINT        | String representing server gRPC endpoint                          | 127.0.0.1:7233 |
 | TEMPORAL_WEB_PORT             | HTTP port to serve on                                             | 8088           |
+| TEMPORAL_MLTS_ROOT            | mTLS root CA file                                                 |                |
```

Comment:
Is it possible to have the same naming as temporal?

Reply:
yes, updated